web-registry/src/pages/Marketplace/Storefront/hooks/useBuySellOrderSubmit.tsx

```diff
@@ -62,7 +101,7 @@ const useBuySellOrderSubmit = ({
             value: { name: batchDenom, url: `/credit-batches/${batchDenom}` },
           },
           {
-            label: 'amount retired',
+            label: disableAutoRetire ? 'NUMBER OF CREDIT' : 'amount retired',
```

Comment:
can't find this in the mockups - should it be
```suggestion
            label: disableAutoRetire ? 'NUMBER OF CREDITS' : 'amount retired',
```
?